src/app/public/modules/lookup/lookup-autocomplete-adapter.ts

```diff
@@ -12,30 +12,79 @@ import {
 } from '../autocomplete/types/autocomplete-search-function-filter';
 
 export class SkyLookupAutocompleteAdapter {
+
+/**
+ * Specifies a data source for the lookup component to search when users
+ * enter text. You can specify static data such as an array of objects, or
+ * you can pull data from a database.
+ * @default []
+ */
   @Input()
   public data: any[];
 
+/**
+ * Specifies how many milliseconds to wait before searching while users
+ * enter text in the lookup field.
+ * @default 0
+ */
   @Input()
   public debounceTime: number;
 
+/**
+ * Specifies an object property to display in the text input after users
+ * select an item in the dropdown list.
+ * @default name
+ */
   @Input()
   public descriptorProperty: string;
 
+/**
+ * Specifies an array of object properties to search.
+ * @default ['name']
+ */
   @Input()
   public propertiesToSearch: string[];
 
+/**
+ * Specifies a function to dynamically manage the data source when users
+ * change the text in the lookup field. The search function must return
+ * an array or a promise of an array. The `search` property is particularly
+ * useful when the data source does not live in the source code.
+ */
   @Input()
   public search: SkyAutocompleteSearchFunction;
```

Comment:
The output type `SkyAutocompleteSearchFunction` isn't showing on this docs page, but I'm pretty sure its related to this issue here: https://github.com/blackbaud/skyux-docs-tools/issues/125. Doesn't need to block your PR, just noting it.